pkg/services/secrets/kvstore/plugin_mig_test.go

```diff
@@ -89,6 +91,7 @@ func setupTestMigratorService(t *testing.T) (*PluginSecretMigrationService, Secr
 		sqlStore,
 		secretsService,
 		remoteCheck,
+		notifier.NewFakeKVStore(t),
```

Comment:
I find it a bit weird we are using the `NewFakeKVStore` from `ngalert/notifier`, maybe `infra/kvstore` should be the one providing that function so it can be used in multiple services for testing. 

But this could be a future refactor unrelated to this PR but worth noting.

Reply:
You could also instantiate a test kvstore like so:
```
		sqlStore := sqlstore.InitTestDB(t)
		kvStore := kvstore.ProvideService(sqlStore)
```

Reply:
yeah, I like the second way, will change it